### ISSUE TITLE
Nametag Edit (Display Problem)

### ISSUE THREAD

**Issue by MinerMends** [sgtcaze/NametagEdit#820]
Type /ver NametagEdit and post the output.
Latest release from Spigot

What version of Bukkit/Spigot/PaperSpigot are you using? Type /ver

This server is running Paper version git-Paper-270 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: e22dacf)
You are running the latest version

What plugins are you using? Type /plugins
Plugins (61): ajLeaderboards, AntiCooldown, AuctionHouse, AutoSmelt, Banknotes*, BedrockPlayerManager, BetterDeathScreen, BetterRTP, BisectHosting, Blocks_To_Commands*, ccInstantRespawn*, ChatItem, Citizens, CitizensGUI, ClicksPerSecond, CommandDoesNotExist, DecentHolograms, Enterprise, Essentials, EssentialsChat*, ExcellentCrates, EzHomes, Factions, floodgate, Geyser-Spigot, GroundItems, Images, LaggRemover, LegacyKB*, LiteEco, mcMMO, Multiverse-Core, NametagEdit, NexEngine, NoteBlockAPI, ObsidianBreaker*, ore-control, Orebfuscator, PermissionsEx*, PlaceholderAPI, PlayerKits, PlayerVaults, ProtocolLib, PVPControl, ServerListMotdSpigot, SetSpawn, ShopGUIPlus, SilkSpawners, SimpleGeyserVL, SimpleScore, SimpleTpa*, SmashHit*, StackMob, TabList, TradeSystem, Trading*, TubeTils, Vault, VoidGen, WorldEdit, WorldGuard

Please explain your issue. How do you trigger it?

We have a problem on Java / Bedrock. When a user joins their nametag will display but it sometimes doesn't display anything only their name in white text. It takes a bit for it to actually adjust back to the normal tag. Everytime the plugin "refreshes" it fixes. Sometimes they have to rejoin to to fix it. This is happening on all clients. It's also not formatting tablist correctly. And yes it's only this plugin as others work.  

Are there any errors in the console? Please use: https://pastebin.com/
Nope

**Comment by PanetiX** [comment]
Hello, we have the same issue. When users change the worlds the will lose the nametag (sometimes)

**Comment by MinerMends** [comment]
Yeah assuming this is just a broken plugin then lol. Not sure. It isn’t the
end of the world.

**Comment by FedorXYZ** [comment]
same problem here

**Comment by fcdor** [comment]
@PanetiX @MinerMends set RefreshTagOnWorldChange on true in your config.yml

**Comment by PanetiX** [comment]
> @PanetiX @MinerMends set RefreshTagOnWorldChange on true in your config.yml

that seems to help :)
but there is a issue with the /glow effect from CMI. When users has a glow effect, the NameTag will not be displayed.
When i use /nte reload the Tag comes back but the glow color from CMI change to white

**Comment by PanetiX** [comment]
okay that don't helped. Nametage still disappear on world change.
Yes i've set on config.yml "RefreshTagOnWorldChange: true"
